Made simple site search functional

kzadrozny opened the pull request:
Needs @bbusenius review to make it function as expected.

## Brad: I was hoping to have the following indexed:
- News story page: excerpt
- Standard page: Search description

## For news I added this
```
search_fields = Page.search_fields + [
        index.SearchField('excerpt')
]
```
But it doesn't get indexed. Is this a limitation of using the canned Wagtail search?

bbusenius commented:
Yes, I believe this is a limitation. [This is the relevant documentation](https://docs.wagtail.io/en/v2.7/topics/search/backends.html#database-backend-default).

I think that if you change it to something like `MyCustomPageType.search_fields + [` it will probably work, however, we don't want to do that. We should use the [PostgreSQL search engine](https://docs.wagtail.io/en/v2.7/reference/contrib/postgres_search.html#postgres-search) instead. This will be much better and it's still lighter than Elasticsearch.

kzadrozny commented:
## Changes

- Added postgres searching
- Added custom fields to page model indexing
- Fixed small issues with Contact Form and Member Collection search templates

kzadrozny commented on bmrc/templates/streams/memb_coll_search_block.html:
```diff
@@ -1,15 +1,15 @@
 		    {% if self.search_help_text %}
-			    <p class="help" id="coll_help">This is a help text</p>
+			    <p class="help" id="coll_help">{{ value.search_help_text }}</p>
 		    {% endif %}
```
This has been updated in the code and ready for a re-review.
